File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/BlurbApp.java
package com.mowdowndevelopments.blurb;

import android.app.Application;

import timber.log.Timber;

public class BlurbApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Timber.plant(new Timber.DebugTree());
    }
}

File: blurb/app/src/main/java/com/mowdowndevelopments/blurb/BlurbApp.kt
package com.mowdowndevelopments.blurb

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

class BlurbApp : Application() {
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(DebugTree())
        }
    }
}